python/tuir/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                tuir
+version             1.29.0
+supported_archs     noarch
+license             MIT
+
+python.versions     38 39 310
```

Comment:
the `python.version` line is only meaningful if the portname starts with `py-`; in that case the `python` PG will automatically generate the relevant subports. Here you want to set a Python version that should be used and that is done using `python.default_version 310`